andela/dahlia-ah-backend
# #168255060 Random novel and novel of the week

#### Pivotal tracker story
[#168255060](https://www.pivotaltracker.com/story/show/168255060)
#### What does this PR do?
Create endpoint to random novel and novel of the week
#### Summary of Task
- Add random novel endpoint
- Add novel of the week endpoint
- Update genre model with Cover Image and genre color
- Add controllers
- Write tests
- Add Swagger documentation
#### How can this be tested?
- Use **git clone** to clone this repository to local branch
- Use **npm install** to download all necessary packages
- Ensure your database is created or you can use **sequelize db:create** if database does not exist
- Use **npm run test**  to test
- Go your POSTMAN and type **http://localhost:${PORT}/api/v1/novels/random** using a GET request or
- Go your POSTMAN and type **http://localhost:${PORT}/api/v1/noveloftheweek** using a GET request
#### Screenshots (if appropriate)
NA
